docs/metrics.md

```diff
@@ -46,3 +50,24 @@ When you are deploying the build operator in your Kubernetes cluster, you need t
     value: "30,60,90,120,180,240,300,360,420,480"
 [...]
 ```
+
+## Configuration of histogram labels
+
+As the amount of buckets and labels has a direct impact on the number of Prometheus time series, you can selectively enable labels that you are interested in using the `PROMETHEUS_HISTOGRAM_ENABLED_LABELS` environment variable. Use a comma-separated value to enable multiple labels. For example:
+
+```bash
+export PROMETHEUS_HISTOGRAM_ENABLED_LABELS=namespace
+make local
+```
+
+When you are deploying the build operator in your Kubernetes cluster, you need to extend the `spec.containers[0].spec.env` section of the sample deployment file, [operator.yaml](../deploy/operator.yaml), to add an additional entry:
+
+```yaml
+[...]
+  env:
+  - name: PROMETHEUS_HISTOGRAM_ENABLED_LABELS
+    value: namespace
+[...]
+```
+
```

Comment:
Not sure if we need to explicitly mention that only two types of labels are currently supported. Thinking that users might think that more labels are supported, but we know that because of the code, they will not be used.

Reply:
Yes, I agree it is better to tell which label we support now. if default `HistogramEnabledLables` is empty, we should let end user know which label he can add.

Reply:
The supported labels are at the top in the table, anyway I added them to this section as well.